dhis-2/dhis-services/dhis-service-tracker/src/test/java/org/hisp/dhis/tracker/export/event/EventOperationParamsMapperTest.java

```diff
@@ -686,14 +685,8 @@ void shouldMapSearchScopeOrgUnitWhenOuModeAccessible()
     user.setOrganisationUnits(Set.of(searchScopeOrgUnit));
 
     when(currentUserService.getCurrentUser()).thenReturn(user);
-    when(organisationUnitService.getOrganisationUnit(orgUnit.getUid())).thenReturn(orgUnit);
 
-    EventOperationParams requestParams =
-        EventOperationParams.builder()
-            .programUid(program.getUid())
-            .orgUnitUid(orgUnit.getUid())
-            .orgUnitMode(ACCESSIBLE)
```

Comment:
Do we default to mode accessible? You removed it and the test says `shouldMapSearchScopeOrgUnitWhenOuModeAccessible`

Might have been my mistake at some point 🤷🏻 I see the use of `requestParams` name while this class should be `operationparams`. We could replace all with `params` in all the mapper tests then there is no confusion 😅

Reply:
Yes, in this test case we default to ACCESSIBLE.
Good idea to not use requestParams.

Reply:
Not sure why we set the mode in the setup. It is clearer if its set here IMHO